sources/procfs.go

```diff
@@ -194,44 +151,14 @@ func (s *lustreProcfsSource) generateOSTMetricTemplates() error {
 func (s *lustreProcfsSource) generateMDTMetricTemplates() error {
 	metricMap := map[string][]lustreHelpStruct{
 		"mdt/*": {
-			{"md_stats", "open_total", openHelp, s.counterMetric},
-			{"md_stats", "close_total", closeHelp, s.counterMetric},
-			{"md_stats", "getattr_total", getattrHelp, s.counterMetric},
-			{"md_stats", "setattr_total", setattrHelp, s.counterMetric},
-			{"md_stats", "getxattr_total", getxattrHelp, s.counterMetric},
-			{"md_stats", "setxattr_total", setxattrHelp, s.counterMetric},
-			{"md_stats", "statfs_total", statfsHelp, s.counterMetric},
-			{"md_stats", "mknod_total", mknodHelp, s.counterMetric},
-			{"md_stats", "link_total", linkHelp, s.counterMetric},
-			{"md_stats", "unlink_total", unlinkHelp, s.counterMetric},
-			{"md_stats", "mkdir_total", mkdirHelp, s.counterMetric},
-			{"md_stats", "rmdir_total", rmdirHelp, s.counterMetric},
-			{"md_stats", "rename_total", renameHelp, s.counterMetric},
-			{"md_stats", "sync_total", syncHelp, s.counterMetric},
-			{"md_stats", "samedir_rename_total", samedirRenameHelp, s.counterMetric},
-			{"md_stats", "crossdir_rename_total", crossdirRenameHelp, s.counterMetric},
-			{"num_exports", "exports_total", "Total number of times the pool has been exported", s.counterMetric},
-			{"job_stats", "job_open_total", openHelp, s.counterMetric},
-			{"job_stats", "job_close_total", closeHelp, s.counterMetric},
-			{"job_stats", "job_mknod_total", mknodHelp, s.counterMetric},
-			{"job_stats", "job_link_total", linkHelp, s.counterMetric},
-			{"job_stats", "job_unlink_total", unlinkHelp, s.counterMetric},
-			{"job_stats", "job_mkdir_total", mkdirHelp, s.counterMetric},
-			{"job_stats", "job_rmdir_total", rmdirHelp, s.counterMetric},
-			{"job_stats", "job_rename_total", renameHelp, s.counterMetric},
-			{"job_stats", "job_getattr_total", getattrHelp, s.counterMetric},
-			{"job_stats", "job_setattr_total", setattrHelp, s.counterMetric},
-			{"job_stats", "job_getxattr_total", getxattrHelp, s.counterMetric},
-			{"job_stats", "job_setxattr_total", setxattrHelp, s.counterMetric},
-			{"job_stats", "job_statfs_total", statfsHelp, s.counterMetric},
-			{"job_stats", "job_sync_total", syncHelp, s.counterMetric},
-			{"job_stats", "job_samedir_rename_total", samedirRenameHelp, s.counterMetric},
-			{"job_stats", "job_crossdir_rename_total", crossdirRenameHelp, s.counterMetric},
+			{"md_stats", "stats_total", statsHelp, s.counterMetric, true},
```

Comment:
Now that we are changing how these stats are displayed, do we want to keep this `md_stats` as opposed to just `stats`? The `nodeType` label will be `mdt` so we should be able to differentiate this from other `stats`. Mostly putting this here as a mild discussion point, not necessarily because I have an opinion.